core/src/test/java/org/opensearch/sql/expression/text/TextFunctionTest.java

```diff
@@ -377,6 +377,20 @@ void locate() {
         DSL.locate(missingRef, DSL.literal("hello"), DSL.literal(1))));
   }
 
+  @Test
+  void position() {
+    FunctionExpression expression = DSL.position(
+        DSL.literal("world"),
+        DSL.literal("helloworld"));
```

Comment:
can you change this to `"helloworldworld"`?  Then it will also cover the duplicate case.